lib/ws_proxy.rb

```diff
@@ -58,7 +58,7 @@ def start_proxy
       end
     end
     @proxy_port = port
-    { :port => proxy_port, :password => password, :encrypt => @encrypt }
+    { :port => proxy_port, :password => password, :encrypt => Setting[:websockets_encrypt] }
```

Comment:
`@encrypt` was introduced in 508776ea6dfc7defce8a4e466408629f494da0cf so will need a cherry pick to 2.2 and 2.3.